ourjapanlife/findadoc-web
# Upgrade Node

Resolves #125 


# What changed
- update to Node 18

# Testing instructions
Site should build in netlify (check the deploy log that it uses node 18)
GitHub Actions should pass
